packages/veritone-react-common/src/components/MediaPlayer/VideoSource.js

```diff
@@ -59,17 +114,22 @@ export default class VideoSource extends React.Component {
               video.play();
               return;
             })
-            .catch(err => {
+            .catch((err) => {
               console.log('error loading video with shaka player', err);
             });
         };
       } else {
-        this.player.load(streamUri).catch(err => {
+        this.player.load(streamUri).catch((err) => {
           console.log('error loading video with shaka player', err);
         });
       }
       this.setState({
-        streamUri: streamUri
+        streamUri: streamUri,
+      });
+      this.player.addEventListener('error', () => {
```

Comment:
Is there any specific error code or message for signed url expiration? we might just want to handle specific code for signed url expiration.
https://shaka-player-demo.appspot.com/docs/api/tutorial-basic-usage.html
https://shaka-player-demo.appspot.com/docs/api/shaka.util.Error.html)

Reply:
I think the code is 1001. 
![image](https://user-images.githubusercontent.com/50015330/216616115-617fed1c-b35f-4457-98fd-3808f52bf3c4.png)